src/nodes.coffee

```diff
@@ -361,6 +361,19 @@ exports.Base = class Base
       else
         return true if children.replaceInContext match, replacement
 
+  # `findReferencedVars` recurses through the source parse tree to mark every
+  # "scope" parse node (`Root` and `Code`s) with all referenced
+  # (accessed or assigned) variables in that source scope, so that every scope
+  # knows what to avoid when generating new variable names.
+  # Specifically, it takes in an array of `scopeNodes`,
+  # each of which already has a `@referencedVars` attribute that is a `Set`.
+  # For each found reference to an identifier (see `IdentifierLiteral`'s
+  # override for `findReferencedVars`), it adds the name (as a `String`)
+  # to each scope node's `referencedVars` Set.
```

Comment:
Great comment, just updating typography to match (most, if not all) other comments.

```suggestion
  # `findReferencedVars` recurses through the source parse tree to mark every
  # “scope” parse node (`Root` and `Code`s) with all referenced
  # (accessed or assigned) variables in that source scope, so that every scope
  # knows what to avoid when generating new variable names.
  # Specifically, it takes in an array of `scopeNodes`,
  # each of which already has a `@referencedVars` attribute that is a `Set`.
  # For each found reference to an identifier (see `IdentifierLiteral`’s
  # override for `findReferencedVars`), it adds the name (as a `String`)
  # to each scope node’s `referencedVars` Set.
```